op-acceptance-tests/tests/base/rpc_connectivity_test.go

```diff
@@ -0,0 +1,61 @@
+package base
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/system"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/systest"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+// TestRPCConnectivity checks we can connect to L2 execution layer RPC endpoints
+func TestRPCConnectivity(t *testing.T) {
+	systest.SystemTest(t,
+		rpcConnectivityTestScenario(),
+	)
+}
+
+func rpcConnectivityTestScenario() systest.SystemTestFunc {
+	return func(t systest.T, sys system.System) {
+		logger := testlog.Logger(t, log.LevelInfo)
+		logger.Info("Started L2 RPC connectivity test")
+		ctx, cancel := context.WithTimeout(t.Context(), 30*time.Second)
+		defer cancel()
+
+		// Test each L2 chain's execution RPC
+		for chainIdx, l2Chain := range sys.L2s() {
+			chainIdx := chainIdx
+			t.Run(fmt.Sprintf("L2_Chain_%d", chainIdx), func(t systest.T) {
+				require.NotEmpty(t, l2Chain.Nodes(), "L2 chain has no nodes")
+
+				// Get the first node
+				execNode := l2Chain.Nodes()[0]
```

Comment:
Would it be useful to range through all l2Chain.Nodes instead of only checking the first one?

Reply:
Do we expect all nodes to be accessible via RPC?